services/ymax-planner/src/plan-deposit.ts

```diff
@@ -96,14 +98,23 @@ export const handleDeposit = async (
   if (errors.length) {
     throw AggregateError(errors, 'Could not get balances');
   }
+  const makeGasEstimator = prepareGasEstimator(powers);
   const balances = Object.fromEntries(balanceEntries);
   const transfers = planDepositTransfers(amount, balances, targetAllocation);
+  const transferStepsP = Object.entries(transfers).map(([dest, amt]) =>
```

Comment:
This value isn't a promise, it's an array _of_ promises. I've only ever seen the "P" suffix used for the former.
```suggestion
  const transferStepsPromises = Object.entries(transfers).map(([dest, amt]) =>
```